.github/workflows/ci.yml

```diff
@@ -95,9 +72,13 @@ jobs:
     name: Release
 
     runs-on: ubuntu-latest
-    if: needs.setup.outputs.DOING_RELEASE == '1'
+    if: startsWith(github.ref, 'refs/tags/v')
 
-    needs: [setup, build]
+    needs: [build]
+
+    strategy:
+      matrix:
+        llvm_version: ['10.x', '11.x']
```

Comment:
Can't we use YAML references here? (I know, yes…)

Reply:
Github workflows do not support YAML anchors yet: https://github.com/jubianchi/llvm-custom-builds/actions/runs/381596227